adafruit_rgb_display/ssd1331.py

```diff
@@ -1,4 +1,9 @@
+"""A simple driver for the SSD1331-based displays."""
```

Comment:
Testing with SSD1331 failed. However I get the same results with the current version of the driver as well. Multicolor pixellation and a single vertical line that turns the color that the code attempts to fill the entire display. That 1-pixel-wide line will change color if I change the `fill` color in the code. The line is close to the left side, if that matters.